docker-compose.yml

```diff
@@ -1,43 +1,42 @@
 version: '3'
 
+x-restart-policy: &default_restart_policy
+  condition: on-failure
+  delay: 3s
+  max_attempts: 5
+  window: 60s
+
 services:
   redis:
     deploy:
       replicas: 1
       placement:
         constraints:
-          - node.labels.role == opencti-core
-      restart_policy: &default_restart_policy
-        condition: on-failure
-        delay: 3s
-        max_attempts: 5
-        window: 60s
-    image: redis:7.2.4
-
+        - node.labels.role == opencti-core
+      restart_policy: *default_restart_policy
+    image: redis:7.4.2
     volumes:
       - redisdata:/data
-
+      
   nginx:
     image: codeforafrica/cfa-opencti:0.0.6
```

Comment:
Why was an `nginx` image named `cfa-opencti`? I don't think if one goes into Docker hub and see an image named `codeforafrica/cfa-opencti` would be able to guess that it's not actually and `opencti` image but that of `nginx`. Lets name it `nginx-opencti` or something along those lines.